ua-snap/iem-webapp
# Creates interface to let you search by clicking on the map

You need to run the correct API branch locally to get this functionality working.

Things to test:

 - Click somewhere on the map, observe the results you get.  Sometimes the map bounds aren't perfect (polygons go outside of it or it's oddly centered).
 - Hover over polygons and click to go there.  You should see a hover label when you are over a poly.  Clicking to search a place should say "Loading data for [place]...".
 - Community markers also do a hover/click-to-go, but it's hard to "hit" them because they're normally covered by other polygons.  I tried addressing this with clever use of `bringToBack()` to push the polygons to the background but then they were hidden behind another layer and could no longer be clicked!
 - Other rough edges.
